README.md

```diff
@@ -1,34 +1,54 @@
-web3-plugin-template
-===========
+# Web3.js Plugin for Wallet RPC methods
 
-This is a template for creating a repository for web3.js plugin.
+This Web3.js plugin adds support for the following wallet-related RPC methods:
 
-How to use
-------------
+- wallet_addEthereumChain (EIP-3085)
+- wallet_updateEthereumChain (EIP-2015)
+- wallet_switchEthereumChain (EIP-3326)
+- wallet_getOwnedAssets (EIP-2256)
+- wallet_watchAsset (EIP-747)
+- wallet_requestPermissions (EIP-2255)
+- wallet_getPermissions (EIP-2255)
```

Comment:
It seems like this PR only adds support for one of these methods. It may be a good idea to note that the other methods are not yet supported.

Reply:
yes, this is WIP, I'll add them all before the plugin is ready for release